src/core/operations/ROT13.mjs

```diff
@@ -75,7 +77,7 @@ class ROT13 extends Operation {
                     chr = (chr - 97 + amount) % 26;
                     output[i] = chr + 97;
                 } else if (rotNumbers && chr >= 48 && chr <= 57) { // Numbers
-                    chr = (chr - 48 + amount) % 10;
+                    chr = (chr - 48 + numAmount) % 10;
```

Comment:
Given `numAmount` is only used in this case, it might be easier to define it locally in this section, e.g:

```
                } else if (rotNumbers && chr >= 48 && chr <= 57) { // Numbers
                    const numAmount = 10 - (Math.abs(numAmount) % 10);
                    chr = (chr - 48 + numAmount) % 10;
                    output[i] = chr + 48;
                }
```